v4_abacus.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |spec|
                 
                 
                 
-    if false
+    if !Dir.exist?('build/cocoapods/framework/Abacus.framework') || Dir.empty?('build/cocoapods/framework/Abacus.framework')
```

Comment:
Can we revert this?  It causes issue with iOS build

Reply:
wait, if we just have `if false` as the condition, can't we just delete this whole block? seems like it should never hit

Reply:
this file is auto generated, it's simpler to just replace the condition with false